ttnn/cpp/ttnn/operations/conv/conv_transpose2d/conv_transpose2d.cpp

```diff
@@ -102,7 +102,8 @@ Result conv_transpose2d(
     std::array<uint32_t, 2> dilation,
     uint32_t groups,
     std::optional<const ttnn::Tensor> bias_tensor,
-    std::optional<const conv2d::Conv2dConfig> conv_config_)   {
+    std::optional<const conv2d::Conv2dConfig> conv_config_,
+    const std::optional<const MemoryConfig> memory_config   ) {
```

Comment:
does this memory_config need to be added to pybind?

Reply:
Yes. I have added it.